includes/class-amp-travel-cpt.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Class for Travel Custom Post Type.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class AMP_Travel_CTP
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_CTP {
```

Comment:
Looks like a typo, should probably be `_CPT` instead of `_CTP`.

Reply:
oops yes, will fix